index.html

```diff
@@ -275,5 +275,54 @@ <h1>About</h1>
                 </div>
             </div>
         </section>
+
+        <!-- Contact form -->
+        <section>
+            <!-- Photo at the top -->
+            <div class="contain-img1">
+                <img class="photo-2 " src="/assets/image_geometry_1.png" alt="Background photo">
+            </div>
+
+            <!-- Get stared -->
+            <div class="submit-form">
+                <h1 class="main-name">Get started</h1>
+                <p class="main-text">Lorem ipsum dolor sit, 
+                    amet consectetur adipisicing elit. Itaque, alias!
+                </p>
+
+                <!-- Inputs area -->
+                <form class="inputs-group" 
+                action="https://formspree.io/f/mrgjwpka"
+                method="POST"
+                enctype="multipart/form-data" id="contact">
+                    <input type="text"  name="name" placeholder="Your name" class="name-input" maxlength="30" required>
+                    <input type="email" name="email" placeholder="Your email" class="email-input" required>
+                    <input type="text"  name="Additional-text" placeholder="Enter your message here..." class="message-input" maxlength="500" required>
+                    <button type="submit">send</button>
+    
+                    <div class="start-collaboration">
+                            <a href="https://github.com/ErikStoupignan">Start collaboration</a>
+                    </div>
+                </form>
+
+                <!-- Here are the footer section that includes all the icons -->
+                <footer class="footer-section">
+                    <div class="get-my-resume">
+                        <p>Get My Resume</p>
```

Comment:
- [ ] Great job on following the figma template , let's make it perfect by using the right color in the footer here ☝️

You should use `#92DCDC`    

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/177432184-eb11e4b1-8e4f-47d4-ad7f-c25e38c828b3.png)  |  ![image](https://user-images.githubusercontent.com/60618877/177432158-e8c0d706-46bb-4d1e-b61c-0c4c5673785b.png)

Reply:
oh, you're right! I didn't see it. But I'm already fixing the color. 
Thank you for your comment.
 @sumairq